pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -217,6 +219,10 @@ func (p *ConfigurationEditFlags) AddCreateFlags(command *cobra.Command) {
 		"Labels to set for both Service and Revision. name=value; you may provide this flag "+
 			"any number of times to set multiple labels.")
 	p.markFlagMakesRevision("label")
+	command.Flags().Int64Var(&p.TimeoutSeconds, "timeout", config.DefaultRevisionTimeoutSeconds,
+		"Duration in seconds that the request routing layer will wait for a request delivered to a "+""+
+			"container to begin replying")
+	p.markFlagMakesRevision("timeout")
```

Comment:
Double checking the flag, it should probably go into `addSharedFlags` functions to be present in `create, update, apply`.

Reply:
Done